flink-clients/src/main/java/org/apache/flink/client/cli/ProgramOptions.java

```diff
@@ -119,6 +124,14 @@ else if (args.length > 0) {
 		} else {
 			this.savepointSettings = SavepointRestoreSettings.none();
 		}
+
+		if (line.hasOption(JOB_ID_OPTION.getOpt())) {
+			String jobIdRaw = line.getOptionValue(JOB_ID_OPTION.getOpt());
+			UUID id = UUID.fromString(jobIdRaw);
```

Comment:
It's a bit weird that this parameter would use a different format for jobId than the others. I think you should follow the preexisting format.